### PR TITLE
Update danish translation (v.1.1)

### PR DESCRIPTION
**Pull request by Stisen1**
- Updates to Editor-related translations
- Changes to translation of certain in-game words, so that they correspond better with words used in the editor
- Compression of textstrings inside the edit hero properties window

### CODE REVIEW

File: files/lang/dk.po [hunk 1]
```diff
@@ -391,11 +398,12 @@ msgstr "Tropper af %{count} tilknytninger -%{penalty}"
 msgid "Some undead in army -1"
 msgstr "Der er udøde i hæren -1"
 
-#, fuzzy
 msgid ""
 "Default\n"
 "troop"
-msgstr "Standardhandlinger"
+msgstr ""
+"Standard\n"
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po]: @zenseii ? Is the hyphen looking okay to you now? I've changed it from invisible to VISIBLE ;)

File: files/lang/dk.po [hunk 2]
```diff
@@ -3866,65 +3857,65 @@ msgstr "Personlighed:"
...
-msgstr "Standardhandlinger"
+msgstr ""
+"Standard-\n"
+"bygninger"
 
-#, fuzzy
 msgid "Default Army"
-msgstr "Standardhandlinger"
+msgstr ""
+"Standard-\n"
```
Comment (ihhub) [inline files/lang/dk.po:3873]: Can you please check how it looks in the Editor? I believe we don't expect a multi-line text.

File: files/lang/dk.po [hunk 3]
```diff
@@ -3866,65 +3857,65 @@ msgstr "Personlighed:"
...
 msgid "Toggle the use of default buildings. Custom buildings will be reset!"
 msgstr ""
+"Skift brugen af standardbygninger. Brugerdefinerede bygninger vil blive "
+"nulstillet!"
 
-#, fuzzy
 msgid "Default Buildings"
-msgstr "Standardhandlinger"
+msgstr ""
+"Standard-\n"
```
Comment (ihhub) [inline files/lang/dk.po:3895]: Can you please check how it looks in the Editor? I believe we don't expect a multi-line text.

**Review by ihhub — COMMENTED** [review]
Hi @Stisen1 , I put 2 questions here. Can you please take a look at them?

File: files/lang/dk.po [hunk 4]
```diff
@@ -3866,65 +3857,65 @@ msgstr "Personlighed:"
...
+"bygninger"
 
 msgid "Toggle building construction restriction mode."
-msgstr ""
+msgstr "Skift bygningskonstruktionsbegrænsningstilstand."
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:3899]: Google.Translate says that it exists, but I can certaintly change it for better understanding.

File: files/lang/dk.po (hunk 3 above)
Comment (Stisen1) [reply files/lang/dk.po:3895]: I've made several changes to compress textstrings and make 'em fit within textboxes.

image

File: files/lang/dk.po (hunk 2 above)
Comment (Stisen1) [reply files/lang/dk.po]: Checked. It doesn't support multi-line text... So I've removed the hyphen and linebreak altogether.

File: files/lang/dk.po [hunk 5]
```diff
@@ -395,11 +401,12 @@ msgstr "Tropper af %{count} tilknytninger -%{penalty}"
 msgid "Some undead in army -1"
 msgstr "Der er udøde i hæren -1"
 
-#, fuzzy
 msgid ""
 "Default\n"
 "troop"
-msgstr "Standardhandlinger"
+msgstr ""
+"standard\n"
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:408]: image

 There is not enough room for the capital S. Not in all places with the word "Standard" that is... So I rather not have capital S'es here, but not there

File: files/lang/dk.po [hunk 6]
```diff
@@ -5801,22 +5780,24 @@ msgid ""
 "Default\n"
 "value"
 msgstr ""
+"standard\n"
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:5783]: image

Not enough room for capital S'es

File: files/lang/dk.po [hunk 7]
```diff
@@ -5801,22 +5780,24 @@ msgid ""
...
 msgid "View %{skill} Info"
 msgstr "Vis info om %{skill}"
 
-#, fuzzy
 msgid ""
 "Default\n"
 "skill"
-msgstr "Forsvar"
+msgstr ""
+"standard\n"
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:5799]: no room

File: files/lang/dk.po [hunk 8]
```diff
@@ -5801,22 +5780,24 @@ msgid ""
...
+"værdi"
 
-#, fuzzy
 msgid "Set %{skill} Skill"
-msgstr "Indstil angrebsevne"
+msgstr "Indstil %{skill} Evne"
 
 msgid "Set %{skill} base value. Right-click to reset all skills to default."
-msgstr ""
+msgstr "Indstil %{skill} startværdi. Højreklik: Nulstil evner til standard."
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:5790]: Not enough room for the whole thing...

image

File: files/lang/dk.po [hunk 9]
```diff
@@ -4089,18 +4076,16 @@ msgid "Quit out of the map editor."
...
+msgstr "Indstil radius: Tilfældigt Ultimativt Artefakt"
 
 msgid "%{object} has no properties to change."
 msgstr ""
 
-#, fuzzy
-msgid "This artifact"
-msgstr "med 1 artefakt."
+msgid "Monsters cannot be placed on water."
+msgstr "Væsener kan ikke placeres på vand."
```
[the remark this answers is not in the document]
Comment (ihhub) [reply files/lang/dk.po:4088]: Hi @zenseii , we can still proceed with this. Our pipeline will correct all these either missing or additional strings.

File: files/lang/dk.po [hunk 10]
```diff
@@ -4430,103 +4414,17 @@ msgstr "Intet kort tilgængelig!"
...
-#, fuzzy
-msgid ""
-"Click to set map difficulty. More difficult maps might include more or "
-"stronger enemies, fewer resources, or other special conditions making things "
-"tougher for the human player."
-msgstr ""
-"Det nuværende kort-niveau for det valgte kort. Kortets sværhedsgrad er "
-"udelukkende bedømt af scenariets designer. Vanskeligere kort inkluderer "
-"flere og stærkere fjender, færre resurser eller andre særlige vilkår, som "
-"gør det sværere for menneskelige spillere."
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:4514]: Done

File: files/lang/dk.po [hunk 11]
```diff
@@ -4115,10 +4100,12 @@ msgstr "Der kan maksimalt placeres %{count} helte af samme farve på kortet."
 msgid "Failed to update player information."
 msgstr "Opdatering af spilleroplysninger mislykkedes."
 
-#, fuzzy
-msgid "Only one Random Ultimate Artifact can be placed on the map."
+msgid "Artifacts cannot be placed on water."
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:4089]: Done

File: files/lang/dk.po (hunk 9 above)
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:4074]: Removed it

**Review by ihhub — APPROVED** [review]

**Comment by ihhub** [comment]
@Stisen1 , many thanks for this update!

**Pull request merged**